code/datums/emergency_calls/unitologi.dm

```diff
@@ -0,0 +1,44 @@
+/datum/emergency_call/unitologiststeam
+	name = "Unitologist"
+	probability = 25
+	landmark_tag = "unitologiststeam"
+	specials_outfits = list(/decl/hierarchy/outfit/healer, /decl/hierarchy/outfit/mechanic, /decl/hierarchy/outfit/faithful)
```

Comment:
five different unitologist outfits were already defined in another file

Reply:
> five different unitologist outfits were already defined in another file

Already used.